README.md

```diff
@@ -12,33 +12,76 @@ All local secrets are loaded from a `.env` file in the root directory of the pro
 Those are (example values):
 
 ```.env
+COOKIE_DOMAIN="siasky.net"
+COOKIE_HASH_KEY="any thirty-two byte string is ok"
+COOKIE_ENC_KEY="any thirty-two byte string is ok"
 SKYNET_DB_HOST="localhost"
 SKYNET_DB_PORT="27017"
 SKYNET_DB_USER="username"
 SKYNET_DB_PASS="password"
-COOKIE_DOMAIN="siasky.net"
-COOKIE_HASH_KEY="any thirty-two byte string is ok"
-COOKIE_ENC_KEY="any thirty-two byte string is ok"
 STRIPE_API_KEY="put-your-key-here"
 STRIPE_WEBHOOK_SECRET="put-your-secret-here"
 ```
 
 There are some optional ones, as well:
 
 ```.env
-SKYNET_ACCOUNTS_LOG_LEVEL=trace
-KRATOS_ADDR=localhost:4433
+ACCOUNTS_JWKS_FILE="/accounts/conf/jwks.json"
```

Comment:
cc @firyx 
we will need to update ansible/lastpass for the server set up scripts. We can wait until it is merged but wanted to call your attention to it.